src/css/custom.css

```diff
@@ -231,8 +231,13 @@ table, tr, td, th {
 
 /*  for emoji's */
 
-@media screen and (min-width: 997px) and (max-width: 1119px) {
+@media screen and (min-width: 997px) {
```

Comment:
by default it is `display: none`, then you don't need `@media` at all.

Reply:
can you see this like